checker/raw_result.go

```diff
@@ -25,29 +25,27 @@ import (
 
 // RawResults contains results before a policy
 // is applied.
-//
-//nolint:govet
 type RawResults struct {
-	BinaryArtifactResults       BinaryArtifactData
+	Metadata                    MetadataData
 	BranchProtectionResults     BranchProtectionsData
-	CIIBestPracticesResults     CIIBestPracticesData
-	CITestResults               CITestData
-	CodeReviewResults           CodeReviewData
+	PinningDependenciesResults  PinningDependenciesData
+	SecurityPolicyResults       SecurityPolicyData
```

Comment:
So I know this was alphabetized recently to find the data easier.
https://github.com/ossf/scorecard/pull/3540

Reply:
https://github.com/ossf/scorecard/pull/3791#discussion_r1456296660